pkg/userdata/generate/generate.go

```diff
@@ -64,28 +65,53 @@ type TrustdInfo struct {
 	Token string
 }
 
-// RandomString returns a string of length n.
-func RandomString(n int) string {
-	var letter = []rune("abcdefghijklmnopqrstuvwxy0123456789")
+// randBytes returns a random string consisting of the characters in
+// validBootstrapTokenChars, with the length customized by the parameter
+func randBytes(length int) string {
 
-	b := make([]rune, n)
-	for i := range b {
-		b[i] = letter[rand.Intn(len(letter))]
+	// validBootstrapTokenChars defines the characters a bootstrap token can consist of
+	const validBootstrapTokenChars = "0123456789abcdefghijklmnopqrstuvwxyz"
+
+	// len("0123456789abcdefghijklmnopqrstuvwxyz") = 36 which doesn't evenly divide
```

Comment:
another (weird) way I thought about: use https://godoc.org/crypto/rand#Int with max value of `36 ** length`, so our token is just a number in 36-base system which we encode back into letters

in fact, we can encode via https://godoc.org/math/big#Int.Text with base 36

Reply:
probably it just needs leading `0` padded to the left in case it has less chars than we need